docs/jwst/tweakreg/README.rst

```diff
@@ -428,6 +433,14 @@ in the ``assign_wcs`` step.
 
 * ``sip_npoints``: Number of points for the SIP fit. (Default=12).
 
+**stpipe general options:**
+
+* ``output_use_model``: A boolean indicating whether to use `DataModel.meta.filename`
+  when saving the results. (Default=True)
+
+* ``on_disk``: A boolean indicating whether to keep models in temporary files on disk
```

Comment:
decided (for now) to just change every option in jwst to use `in_memory` since it's already in use for certain steps.  But this could still be changed

Reply:
This looks to now be resolved with all steps using `in_memory`.